modular_zubbers/code/modules/storyteller/gamemode.dm

```diff
@@ -213,7 +213,9 @@ SUBSYSTEM_DEF(gamemode)
 /datum/controller/subsystem/gamemode/proc/get_antag_cap()
 	if(isnull(storyteller))
 		return 0
-	return round(get_correct_popcount() / storyteller.antag_divisor) + (sec_crew ? sec_crew : ANTAG_CAP_FLAT) // Population divide by storyeteller's divisor, add one antag per sec.
+	if(storyteller.antag_divisor == 0)
+		return 0
+	return CEILING( clamp(get_correct_popcount() / storyteller.antag_divisor + sec_crew,ANTAG_CAP_FLAT,sec_crew*1.5), 1 )
```

Comment:
I'm not sure whether to trust ceiling in this, we used it before and for some reason it clamped the whole thing to one even though it makes no sense for it to do so. Whatever, we'll see in testing

```suggestion
	return CEILING(clamp(get_correct_popcount() / storyteller.antag_divisor + sec_crew, ANTAG_CAP_FLAT, sec_crew*1.5), 1)
```